backend/eventd/logger_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
This should be covered by `// +build !windows`, is there a reason to add another build constraint?

Reply:
Hm, you'll have to chat with my editor about this one.

Looks like gofmt did this on my behalf.  https://go.dev/doc/go1.17#gofmt. Striking it for now.

Reply:
@c-kruse I'm seeking clarification of this behaviour on the gophers slack, for future fmt adventures :+1: